scrapers/SofieMariexxx.yml

```diff
@@ -0,0 +1,31 @@
+name: SofieMarie
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sofiemariexxx.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //span[@class="update_title"]
+      Date:
+        selector: //span[@class="availdate"]/text()[1]
+        postProcess:
+          - parseDate: 01/02/2006
+      Performers:
+        Name: //span[@class="tour_update_models"]/a
+      Details:
+        selector: //span[@class="latest_update_description"]
+      Tags:
+        Name: //span[@class="update_tags"]/a
+      Image:
+        selector: //img[@class='stdimage large_update_thumb left thumbs']/@src
+        postProcess:
+          - replace:
+              - regex: ^
+                with: "https://sofiemariexxx.com/"
+      Studio:
+        Name:
+          selector: //meta[@name="author"]/@content
```

Comment:
the studio selector doesnt seem to match anything
any reason a fixed one wasnt used?

Reply:
Ermmmm nope no idea why I did not have that as fixed. It is now though :-)